project_task_digitized_signature/tests/test_project_task_digitized_signature.py

```diff
@@ -11,24 +11,12 @@ def setUpClass(cls):
         cls.task = cls.env["project.task"].create({"name": "Task test #1"})
 
     def test_task_digitized_signature(self):
-        search_domain = [
```

Comment:
This test is now useless, as it's not the testing anything related to the module.